foam_internalwrapper/cuba_extension.py

```diff
@@ -0,0 +1,20 @@
+""" Provisional CUBA keywords specific for this revision
```

Comment:
@jmarcelogimenez , this comment has been not addressed.  An issue should be opened.